source/docs/appendices/migration-guides.md

```diff
@@ -20,6 +20,16 @@ These new features may be relevant for your existing application:
 - [A quickorder module has been created to order by SKU](/docs/advanced/features/quickorder.html)
 - Front-Commerce dependencies are now regularly (and automatically) updated using [Depfu](https://depfu.com/). Patches are automatically applied if the CI is green, minor versions are manually approved. We always review changelogs provided by Depfu. **You can review updates any time [by filtering Merge Requests tagged `depfu`](https://gitlab.com/front-commerce/front-commerce/-/merge_requests?scope=all&state=merged&label_name[]=depfu).**
 
+### Storybook 6
+
+Your styleguide is now powered by Storybook 6. 
+
+This induces several changes in your custom stories.
+
+- The organization of stories is now only manageable with `/`, please set your story name like this : `section/path/of/your/story/Component`
+- As knobs are now deprecated, we highly encourages you to use controls from now on. see [the official documentation for more details](https://storybook.js.org/addons/@storybook/addon-knobs)
+- The way to import markdown files in stories has changed, please move from `import readme from "./readme.md";` to `import { html as readme } from "./readme.md";`
```

Comment:
Now that we have codemods tooling in place. Wouldn't it be something quite fast to implement? So that developers would only have to run one command during the migration.

A `sed` command one-line could also makes the job here though…

I have no idea of how many instances we had in our codebase and if that was tedious or not.

Also, for such changes we usually use `diff` syntax to make it easier to scan and understand by developers (we got positive feedbacks for this practice):
```suggestion
- The way to import markdown files in stories has changed, please update your code as below:
```diff
- import readme from "./readme.md";
+ import { html as readme } from "./readme.md";
```
```

Reply:
import change added to a codemod (usage specified in the migration guide)